airbyte-webapp/src/scss/global.scss

```diff
@@ -0,0 +1,25 @@
+@use "./colors";
+@use "./fonts";
+
+html,
```

Comment:
:information_source: In the styled-component variant this still had a `#__next` on it, which I assume (given we have 0 elements with that id) was a leftover from some very early versions of the app.